lib/tasks/data_maintenance.rake

```diff
@@ -36,4 +36,19 @@ namespace :data_maintenance do
     Person.where(criminal_records_office: '').update_all(criminal_records_office: nil)
     Person.where(police_national_computer: '').update_all(police_national_computer: nil)
   end
+
+  desc 'fix generic event classification for existing data'
+  task fix_generic_event_classifications: :environment do
```

Comment:
```ruby
GenericEvent::PerMedicalAid.update_all(classification: 'medical')

critical_event_types = [
  'GenericEvent::PersonMoveRoadTrafficAccident',
  'GenericEvent::PersonMovePersonEscaped',
  'GenericEvent::PersonMoveUsedForce',
  'GenericEvent::PersonMoveMajorIncidentOther',
  'GenericEvent::PersonMoveSeriousInjury',
  'GenericEvent::PersonMoveMinorIncidentOther',
  'GenericEvent::PersonMoveDeathInCustody',
  'GenericEvent::PersonMoveAssault',
  'GenericEvent::PersonMovePersonEscapedKpi',
  'GenericEvent::PersonMoveReleasedError',
]

GenericEvent.where(type: types).update_all(classification: 'critical')
```

Reply:
👍

Reply:
Running this task may be tricky on prod - you'll be updating 400k rows. Best to test it on pre-prod first.